### PR TITLE
SolitareStation FIXES some access problems

### PR DESCRIPTION
**Pull request by SmoSmoSmoSmok**
## About The Pull Request

chaplains can access morgue
chaplains and curators can access service maints and service lathe
Moved 2 pipes 1 tile up to connect loose ends

## Why It's Good For The Game

t

## Changelog

:cl:
add: Added new mechanics or gameplay changes
add: Added more things
expansion: Expands content of an existing feature
del: Removed old things
qol: made something easier to use
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

File: _maps/map_files/SolitaireStation/Solitairestation.dmm [hunk 1]
```diff
@@ -64099,7 +64115,8 @@
 /area/command/gateway)
 "tqQ" = (
 /obj/machinery/door/airlock/maintenance{
-	req_access_txt = "12"
+	name = "Service Maintenance Access";
+	req_one_access_txt = "12;73"
```
Comment (JohnFulpWillard) [inline _maps/map_files/SolitaireStation/Solitairestation.dmm:64119]:
```suggestion
	req_one_access_txt = "73"
```

File: _maps/map_files/SolitaireStation/Solitairestation.dmm [hunk 2]
```diff
@@ -46411,7 +46428,8 @@
 /area/space)
 "nbV" = (
 /obj/machinery/door/airlock/maintenance{
-	req_access_txt = "12"
+	name = "Service Maintenance Access";
+	req_one_access_txt = "12;73"
```
Comment (JohnFulpWillard) [inline _maps/map_files/SolitaireStation/Solitairestation.dmm:46432]:
```suggestion
	req_one_access_txt = "73"
```

File: _maps/map_files/SolitaireStation/Solitairestation.dmm [hunk 3]
```diff
@@ -22535,6 +22537,19 @@
 /obj/machinery/holopad,
 /turf/open/floor/iron,
 /area/tcommsat/server)
+"giX" = (
+/obj/machinery/door/airlock/medical{
+	name = "Morgue";
+	req_one_access_txt = "6;5;28"
```
Comment (JohnFulpWillard) [inline _maps/map_files/SolitaireStation/Solitairestation.dmm:22543]:
```suggestion
	req_one_access_txt = "6"
```
Chefs have morgue access

File: _maps/map_files/SolitaireStation/Solitairestation.dmm (hunk 1 above)
Comment (JohnFulpWillard) [reply _maps/map_files/SolitaireStation/Solitairestation.dmm:64119]: SERVICE SHOULD BE FOR SERVICE! F EVERYONE ELSE

File: _maps/map_files/SolitaireStation/Solitairestation.dmm (hunk 3 above)
Comment (SmoSmoSmoSmok) [reply _maps/map_files/SolitaireStation/Solitairestation.dmm]: ya sure dude this is what i get for using helio sation as a refrnce

**Pull request merged**